xdsl/ir/core.py

```diff
@@ -672,28 +707,6 @@ def create(
         Operation.__init__(op, operands, result_types, attributes, successors, regions)
         return op
 
-    def replace_operand(self, operand: int | SSAValue, new_operand: SSAValue) -> None:
```

Comment:
Maybe deprecate for the next release? I don't care too much, but people downstream might be happy :wink: